tedana/decomposition/pca.py

```diff
@@ -282,7 +282,12 @@ def tedpca(data_cat, data_oc, combmode, mask, adaptive_mask, t2sG,
 
     # Save component table and associated json
     temp_comptable = comptable.set_index("Component", inplace=False)
-    io_generator.save_file(temp_comptable, "PCA metrics tsv")
+    temp_comptable.to_csv(
+        io_generator.get_name("PCA metrics tsv"),
+        index=True,
+        index_label="Component",
+        sep='\t',
+    )
```

Comment:
`io_generator.save_file` doesn't keep the "Component" index.

Reply:
We may want to add that functionality instead.

Reply:
I think it's fixed if we just don't set the index as Component, so I'll test that out now.

Reply:
Everything looks good now.